src-java/grpc-speaker/grpc-service/src/main/java/org/openkilda/grpc/speaker/client/GrpcSession.java

```diff
@@ -290,7 +290,7 @@ public CompletableFuture<Optional<CliReply>> setConfigRemoteLogServer(RemoteLogS
         GrpcResponseObserver<CliReply> observer = new GrpcResponseObserver<>(address, SET_CONFIG_REMOTE_LOG_SERVER);
 
         RemoteLogServer logServer = RemoteLogServer.newBuilder()
-                .setIpaddr(remoteServer.getIpAddress())
+                .setIpaddr(verifyHostAddress(remoteServer.getIpAddress()))
```

Comment:
I've noticed this is not being tested. Could you please test at least the static method?

Reply:
hi @pablomuri added tests